doc/source/ecosystem.rst

```diff
@@ -27,7 +27,7 @@ substantial projects that you feel should be on this list, please let us know.
 Statistics and Machine Learning
 -------------------------------
 
-`Statsmodels <http://www.statsmodels.org/>`__
+`Statsmodels <https://www.statsmodels.org/>`__
```

Comment:
i think you have to make this the same length

Reply:
Not sure what you mean. These are rendering okay and hyperlink to the correct page when I run it locally.

<img width="748" alt="screen shot 2018-11-04 at 9 11 04 am" src="https://user-images.githubusercontent.com/10647082/47967408-c2ad4000-e011-11e8-9d08-c6b06f4d4827.png">

Reply:
they are doc warnings on building i think

Reply:
Gotcha. I see them. Will fix and push on green